infra/notifications/src/emailSend.js

```diff
@@ -19,10 +18,140 @@ if (!process.env.SENDGRID_FROM_EMAIL) {
   logger.warn('Warning: SENDGRID_FROM_EMAIL env var is not set')
 }
 
+const {
+  getMessageFingerprint,
+  isNotificationDupe,
+  logNotificationSent
+} = require('./dupeTools')
+
+//
+// Email notifications for Messages
+//
+async function messageEmailSend(receivers, sender, config) {
+  if (!receivers) throw 'receivers not defined'
+  if (!sender) throw 'sender not defined'
+
+  // Load email template
+  const templateDir = `${__dirname}/../templates`
+
+  // Standard template for HTML emails
+  const emailTemplateHtml = _.template(
+    fs.readFileSync(`${templateDir}/emailTemplate.html`).toString()
+  )
+  // Standard template for text emails
+  const emailTemplateTxt = _.template(
+    fs.readFileSync(`${templateDir}/emailTemplate.txt`).toString()
+  )
+
+  const messageSender = Identity.findOne({
+    raw: true,
```

Comment:
What's the reason for setting raw to true ? 
Afaik we don't use that in the rest of the codebase...

If it is really necessary, maybe add a comment explaining why.
It was not obvious to me reading the code :)

Reply:
This is me being new to Sequelize and trying to console.log() stuff for debugging. Removed in 977111f19e8d5e38162b1879b3ba345a4ae22859.